### PR TITLE
Add archway constantine-3 and archway-1 networks

### PR DESCRIPTION
**Pull request by adairrr**
This change adds Archway's new testnet and mainnet networks.

### CODE REVIEW

File: cw-orch/src/daemon/networks/archway.rs [hunk 1]
```diff
@@ -17,3 +20,29 @@ pub const CONSTANTINE_1: ChainInfo = ChainInfo {
...
+    fcd_url: None,
+};
+
+/// Archway Docs: <https://docs.archway.io/resources/networks>
+/// Parameters <https://www.mintscan.io/archway/parameters>
+pub const ARCHWAY_1: ChainInfo = ChainInfo {
+    kind: ChainKind::Mainnet,
+    chain_id: "archway-1",
+    gas_denom: "aarch",
+    gas_price: 1000000000000.0,
```
Comment (Buckram123) [inline cw-orch/src/daemon/networks/archway.rs:43]: Interesting, it's 900000000000 rn: https://api.mainnet.archway.io/archway/rewards/v1/estimate_tx_fees How often do they update it? Hopefully we can have it as constant

File: cw-orch/src/daemon/networks/archway.rs [hunk 2]
```diff
@@ -6,7 +6,10 @@ pub const ARCHWAY_NETWORK: NetworkInfo = NetworkInfo {
     coin_type: 118u32,
 };
 
-/// <https://docs.archway.io/docs/overview/network>
+#[deprecated(
+    since = "0.6.1",
+    note = "Constantine-1 does not exist anymore. Use Constantine-3 instead."
+)]
```
Comment (Buckram123) [inline cw-orch/src/daemon/networks/archway.rs:12]: nice

File: cw-orch/src/daemon/networks/mod.rs [hunk 3]
```diff
@@ -12,7 +12,7 @@ pub mod sei;
 pub mod terra;
 
 pub use crate::daemon::chain_info::{ChainInfo, ChainKind, NetworkInfo};
-pub use archway::CONSTANTINE_1;
+pub use archway::{ARCHWAY_1, CONSTANTINE_1, CONSTANTINE_3};
```
Comment (Buckram123) [inline cw-orch/src/daemon/networks/mod.rs:15]:
```suggestion
pub use archway::{ARCHWAY_1, CONSTANTINE_3};
```
deprecated

File: cw-orch/src/daemon/networks/mod.rs [hunk 4]
```diff
@@ -40,6 +40,8 @@ pub fn parse_network(net_id: &str) -> ChainInfo {
         LOCAL_TERRA,
         INJECTIVE_888,
         CONSTANTINE_1,
```
Comment (Buckram123) [inline cw-orch/src/daemon/networks/mod.rs:42]:
```suggestion
```

**Review by Buckram123 — APPROVED** [review]
Nice

File: cw-orch/src/daemon/networks/archway.rs (hunk 1 above)
Comment (adairrr) [reply cw-orch/src/daemon/networks/archway.rs:43]: That's the minimum gas price, according to the chain-registry: https://github.com/cosmos/chain-registry/blob/master/archway/chain.json#L20-L23

**Pull request merged**